code/modules/lore_codex/news_data/main.dm

```diff
@@ -1310,3 +1349,140 @@
+	<br><br>\
+	The cruise liner itself has suffered severe damage to the aft port nacelle, as well as notable damage in several other places along the port side, including numerous passenger compartments. With preliminary critical hull repairs complete, the vessel is reported to be in good spirits, and is expected to make a full recovery.\
+	<br><br>\
+	Emergency accommodation for passengers has been organized at the Grand Tack Hotel on the NCS Northern Star, ring three, segment five. All inquiries should be directed to the Emergency Response inbox \[at this address\]. Due to the number of urgent cases, personal inquiries are currently discouraged. NanoTrasen's Emergency Response Team will be contacting appropriate parties over the coming days as the situation is resolved.\
+	<br><br>\
+	\[Note: An earlier version of this story incorrectly identified the passenger count as 200, this number referred to still-missing ship crew.\]"
+
+/datum/lore/codex/page/article129
+	name = "08/07/75 - Apostate Hegemony Escort Poised To Picket Gavel's Zaddat"
+	data = "The ex-Hegemony warship known as IDV All The Stars' Reaching has publicly announced its intent to transit to the Gavel system in advance of the incoming Hegemony fleet. The ship, previously stationed in the nearby system of Kess-Gendar, first came to SolGov space as part of the Zaddat Escort Fleet in 2550 but broke with Hegemony command after disobeying orders to abandon the zaddat Colony Intrepid in 2566.\
+	<br><br>\
+	\"Our change in station has no relation to the Moghes Hegemony's upcoming military excursion,\" the ship's captain, Assagra Ehssor of the heretic Usehi clan, clarified. \"The safety of the Colonies are our sacred duty, and we mean only to provide protection to our defenseless charges in light of Gavel's recent spate of pirate attacks.\"\
+	<br><br>\
+	Gavel is home to a sizable population of Unity heretics, as well as the zaddat Colony Itinerant, whose legal status following the withdrawal of the Hegemony during the Incursion has never been properly clarified. While the ship has already spent several months in Gavel in late '72, some analysts view this as a naked act of provocation against the Hegemony, which the zaddat of Colony Itinerant still consider their patron government.\
+	<br><br>\
+	\"This is absolutely a religious issue for them,\" said Kaijo Day, Unity Studies professor at Elysium University. \"Heretic ideologies have been steadily gaining traction among the so-called loyalist zaddat since the end of the Incursion, and they fear that the Hegemony fleet will make some attempt to convert Colony Itinerant back to orthodoxy, or even to assume direct control of it. In that light, their movements should be understood as an immediate threat towards Hegemony missionaries.\"\

```

Comment:
```suggestion
	\"This is absolutely a religious issue for them,\" said Kaijo Day, Unity Studies professor at Elysium University. \"Heretic ideologies have been steadily gaining traction among the so-called loyalist zaddat since the end of the Incursion, and they fear that the Hegemony fleet will make some attempt to convert colony Itinerant back to orthodoxy, or even to assume direct control of it. In that light, their movements should be understood as an immediate threat towards Hegemony missionaries.\"\

```